packages/frame-core/src/types.ts

```diff
@@ -40,16 +58,12 @@ export type FrameContext = {
     pfpUrl?: string;
   };
   location?: FrameLocationContext;
```

Comment:
slight preference for explicit global context vs undefined

Reply:
could you check now, called it `launcher` instead of home so it's more generic

Reply:
good call